src/Extensions/ExtHostClient_SCM.re

```diff
@@ -65,9 +62,7 @@ module Decode = {
         `String(tooltip),
         `Bool(strikeThrough),
         `Bool(faded),
-        source,
-        letter,
-        color,
+        ..._additionalArgs,
```

Comment:
Just in case?

Reply:
If I changed it to this: 
```
`List([
        `Int(handle),
        uri,
        `List(icons),
        `String(tooltip),
        `Bool(strikeThrough),
        `Bool(faded),
      ]
```
it wouldn't pattern match the 'v1' protocol.

Reply:
I'll remove it for the 'v2' protocol, though.